rotary/rules/rotary.smk

```diff
@@ -455,8 +473,8 @@ rule polish_polypolish:
     benchmark:
         "benchmarks/{step}/polypolish.txt"
     params:
-        qc_short_r1=config.get("qc_short_r1"),
-        qc_short_r2=config.get("qc_short_r2")
+        qc_short_r1=SHORT_R1,
```

Comment:
One nice thing about adding the short read QC workflow in the future is that it will reduce the number of calls to the SHORT_R1 and SHORT_R2 globals substantially!